packages-exp/auth-exp/demo/rollup.config.js

```diff
@@ -37,28 +63,28 @@ const es5Builds = [
    */
   {
     input: 'src/index.js',
-    output: [{ file: pkg.bundle, format: 'esm', sourcemap: true }],
-    plugins: commonPlugins
-  },
-  {
-    input: 'src/worker/index.ts',
-    output: [{ file: pkg.worker, format: 'esm', sourcemap: true }],
+    output: [{ file: pkg.browser, format: 'esm', sourcemap: true }],
     plugins: [
       ...commonPlugins,
-      typescriptPlugin({
-        typescript,
-        tsconfigOverride: {
-          compilerOptions: {
-            lib: [
-              // TODO: remove this
-              'dom',
-              'es2015',
-              'webworker'
-            ]
-          }
+      resolve({
+        mainFields: ['module', 'main']
+      }),
+      terser({
+        output: {
+          comments: false
         }
       })
     ]
+  },
+  {
+    input: 'src/worker/web-worker.ts',
+    output: [{ file: pkg.webworker, format: 'esm', sourcemap: true }],
+    plugins: workerPlugins
+  },
+  {
+    input: 'src/worker/service-worker.ts',
+    output: [{ file: pkg.serviceworker, format: 'esm', sourcemap: true }],
```

Comment:
How does it work? I don't think esm works in worker context.

Reply:
it works cause we don't have actually have any external dependencies, all the import statements are resolved by the time the output is generated, should I use something else here?

Reply:
ah I see. LGTM